scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -0,0 +1,386 @@
+// Script assets have changed for v2.3.0 see
+// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
+
+
+global.trait_list = {
+	"shitty_luck":{
+		luck:-4
```

Comment:
in the future, it would perhaps be better to define this as :

`add_stat(luck, -4)`

although as a first iteration it's fine

Reply:
no because this trait list is techniacally (while hard coded rn) meant to be json files so mod traits can be made,  so all traits are then added by using the add_trait method of each unit's struct e.g unit.add_trait(trait_name). so essentially what you've said is what's happening

Reply:
ok, now I understand, good idea to prepare for extracting data into json.